tensorrt_llm/_torch/modules/fused_moe.py

```diff
@@ -812,9 +812,16 @@ def load_expert_fc2_input_scale_fp8_qdq(
                                            dtype=torch.float32)
         tmp_fc2_input_scale = torch.empty(self.num_experts, dtype=torch.float32)
         for expert_id in range(self.num_experts):
-            w1_input_scale = weights[f"{expert_id}.w1.input_scale"]
-            w3_input_scale = weights[f"{expert_id}.w3.input_scale"]
-            w2_input_scale = weights[f"{expert_id}.w2.input_scale"]
+            if f"{expert_id}.w1.input_scale" in weights:
+                w1_input_scale = weights[f"{expert_id}.w1.input_scale"]
+                w3_input_scale = weights[f"{expert_id}.w3.input_scale"]
+                w2_input_scale = weights[f"{expert_id}.w2.input_scale"]
+            else:
+                # Hack to load non standard checkpoint format
+                # TODO: Remove it after we switch to standard formart
+                w1_input_scale = weights[f"gate_up_proj_input_scale"]
+                w3_input_scale = weights[f"gate_up_proj_input_scale"]
+                w2_input_scale = weights[f"down_proj_input_scale"]
 
```

Comment:
I added an `MoEWeightLoadingMode` in the previous PR to specify the fused `gate_up_proj` format, can we use that? Seems like it's here to stay